INSTALL

```diff
@@ -21,6 +21,10 @@ WHAT DO YOU NEED ?
 
    - a C compiler
 
+   - an IEEE-754 compliant architecture with rounding to nearest
+     ties to even as default rounding mode (most architectures
+     should work nowadays)
```

Comment:
What happens if the architecture is not compliant?

Reply:
At best the configure phase detects it and aborts. At worst, one may be able to prove False on this architecture. Do you think this is worth mentioning (maybe in a paragraph below)?

Reply:
Yes, absolutely worth mentioning!

Reply:
Hi @Zimmi48 @proux01,
so I've just pushed a commit with one such mention (in the `INSTALL` file):
fbeb6eefb74c18230366d2ef6643b49f72ed21b3